Formula/cava.rb

```diff
@@ -0,0 +1,32 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake"
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "libtool"
```

Comment:
Are `automake` and `libtool` needed at runtime?

Reply:
No. At buildtime.

Probably I misunderstood depends_on there.
Can I savely ignore depends_on for libtool and automake that when building?

Reply:
Sorry, I don't understand your question. Could you rephrase it?